src/ops.rs

```diff
@@ -279,7 +286,12 @@ impl SpongeEncryptable for Message {
         {
             Ok(())
         } else {
+            xor_bytes(&mut self.msg, &m);
```

Comment:
looks great! this is the meat of the whole thing and really all it takes to fix a bad password